test/battle/ability/sheer_force.c

```diff
@@ -1,14 +1,20 @@
 #include "global.h"
 #include "test/battle.h"
 
+
+ASSUMPTIONS
+{
+    ASSUME(MoveIsAffectedBySheerForce(MOVE_ELECTRO_SHOT) == TRUE);
+}
+
```

Comment:
Why an assumption specifically for Electro Shot?

Reply:
Yup, Electro Shot uses a "hack" to be Sheer Force boosted so it's just a check for that.